ddl/placement/utils.go

```diff
@@ -131,3 +131,26 @@ func BuildPlacementCopyBundle(oldBundle *Bundle, newID int64) *Bundle {
 	}
 	return newBundle
 }
+
+// GetLeaderDCByBundle return the leader's dc by Bundle
+func GetLeaderDCByBundle(bundle *Bundle, dcLabelKey string) string {
+	dc := getRoleDCByBundle(bundle, dcLabelKey, Leader)
+	if len(dc) > 0 {
+		return dc
+	}
+	// If there is placement policy for leader, fallback to search voter
+	return getRoleDCByBundle(bundle, dcLabelKey, Voter)
```

Comment:
It's not right. Multiple rules may exist for voters.
e.g.
```sql
CONSTRAINTS='{"+zone=bj":1,"+zone=sh":2}' ROLE=voter
```
This clause results in 2 rules, one for `bj` and one for `sh`. They are all in the same bundle.

Also, if you want to make it more reasonable, you can consider this situation:
```sql
ALTER PLACEMENT POLICY CONSTRAINTS='["+zone=bj","+zone=sh"]' ROLE=voter,
ALTER PLACEMENT POLICY CONSTRAINTS='["-zone=bj"]' ROLE=follower
```
This indicates that the leader could only be placed in `sh`.